doc/whatsnew/fragments/8603.bugfix

```diff
@@ -0,0 +1,3 @@
+Added escaping of vertical bar character in annotation labels produced by DOT printer to ensure it is not treated as field separator of record-based nodes.
```

Comment:
```suggestion
``pyreverse``: added escaping of vertical bar character in annotation labels produced by DOT printer to ensure it is not treated as field separator of record-based nodes.
```

Reply:
Done in commit 72bb5af492c484e7d6d9a2924992dd6ce969ff08